src/index.html

```diff
@@ -20,8 +21,369 @@
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card main__card"
+        data-qa="card"
+      >
+        <section class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </section>
+
+        <section class="card__product-code">Product code: 195434</section>
+
+        <section class="card__stars stars">
+          <div class="stars__stars-box">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-price">$2,199</div>
+        </section>
+
+        <section class="card__button">
+          <div
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </div>
+        </section>
+      </div>
+      <div
+        class="card main__card"
+        data-qa="card"
+      >
+        <section class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </section>
+
+        <section class="card__product-code">Product code: 195434</section>
+
+        <section class="card__stars stars">
+          <div class="stars__stars-box">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-price">$2,199</div>
```

Comment:
The price is the same for all cards. Ensure that each product card displays the correct price if they are meant to be different products.